phtc/logic_model/views.py

```diff
@@ -1,14 +1,10 @@
-from django.contrib.auth.decorators import login_required
-from django.http import HttpResponse, HttpResponseForbidden
+import simplejson
```

Comment:
Django 1.5 deprecates django.utils.simplejson in favor of Python 2.6’s built-in json module. You can just swap out "simplejson" for "json".
https://docs.djangoproject.com/en/1.8/releases/1.5/#system-version-of-simplejson-no-longer-used